recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ source:
 build:
   noarch: python
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

Comment:
@mwcraig - `spectral-cube` uses the helpers, yet this works. It seems that we no longer need to hack with ``--offline``?